frontend/ui/data/creoptions.lua

```diff
@@ -528,6 +519,39 @@ Note that your selected font size is not affected by this setting.]]),
                     value_step = 1,
                 },
             },
+            {
+                name = "font_base_weight",
+                name_text = _("Font Weight"),
+                toggle = { "-1", "-½", "0", "+½", "+1", "+1½", "+2" },
+                values = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                args = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                default_value = 0,
+                event = "SetFontBaseWeight",
+                more_options = true,
+                more_options_param = {
+                    value_min = -3,
+                    value_max = 5.5,
+                    value_step = 0.25,
+                    precision = "%+.2f",
+                    value_hold_step = 1,
+                },
+                help_text = _([[Set the font weight delta from "regular" to apply to all fonts.
+
+- 0 will use the "regular (400)" variation of a font.
+- 1 will use the "medium (500)" variation of a font if available
+- 3 will use the "bold (700)" variation of a font if available
+If a font variation is not available, and for fractional deltas, a font will be synthesized from the nearest available weight of the font.]]),
+                help_text_func = function(configurable, document)
+                    local font_face = document:getFontFace()
+                    local available_weights = cre.getFontFaceAvailableWeights(font_face)
+                    return T(_("The default font '%1' is available in these real weights: %2"), font_face, table.concat(available_weights, ", "))
```

Comment:
Oh, there I'd possibly use `weight class`, though ;).

Reply:
Should I keep the "real" in there ? in these _real_ weight classes ?

Reply:
I agree that it feels slightly clunky, but I couldn't really think of anything better. Perhaps getting rid of it entirely does the trick, though, the context should make it clear.

Reply:
"The default font 'Noto Sans' is available in 400, 700." ? Feels a bit short without "real", one could think +1 and +2 won't work.

Reply:
But might be OK, above we state "If a font variation is not available, a font variation will be synthesized...".

Reply:
Oh, no, sorry, I meant just omitting "real" ;).

Reply:
Oh :/
Anybody feel free to correct this in his next PR if "available in 400, 700." feels like broken english :)

Reply:
`available in: x, y`?